<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>virement</class>
 <widget class="QDialog" name="virement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_number1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>account_number1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_number2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>account_number2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Banque1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Boursorama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Revolut</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Banque2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Boursorama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Revolut</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Amount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>virement</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>virement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
